### PR TITLE
text alignment issue in blog card has been fixed

### PR DESCRIPTION
**Pull request by KaveenLaksitha**

### CODE REVIEW

File: src/pages/blog/index.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import ScrollToTop from "react-scroll-to-top";
 
 const Blogs: React.FC = () => (
   <div className="container">
-      <ScrollToTop smooth />
+      <ScrollToTop smooth style={{marginBottom:60}}/>
```
Comment (rusiruavb) [inline src/pages/blog/index.tsx:8]: Do not create inline styling. Create a separate class and apply it here.

File: src/pages/board/index.tsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import ScrollToTop from "react-scroll-to-top";
 
 const Boards: React.FC = () => (
   <div className="container">
-    <ScrollToTop smooth />
+    <ScrollToTop smooth style={{marginBottom:60}}/>
```
Comment (rusiruavb) [inline src/pages/board/index.tsx:7]: Do not create inline styling. Create a separate class and apply it here.

**Review by rusiruavb — CHANGES_REQUESTED** [review]
@KaveenLaksitha I have add some comments. Please take a 👀

File: src/pages/about/index.tsx [hunk 3]
```diff
@@ -6,7 +6,7 @@ import ScrollToTop from "react-scroll-to-top";
 
 const About: React.FC = () => (
   <div className="container">
-    <ScrollToTop smooth />
+    <ScrollToTop smooth style={{marginBottom:60}}/>
```
[the remark this answers is not in the document]
Comment (KaveenLaksitha) [reply src/pages/about/index.tsx:9]: Component disappears when you add a className property

[the remark this answers is not in the document]
Comment (miyurugunarathna) [reply src/pages/about/index.tsx:9]: @KaveenLaksitha I think when you use className property you have to override those styles. Please take a look.
https://github.com/HermanNygaard/react-scroll-to-top/blob/master/src/styles.css
image

**Review by miyurugunarathna — CHANGES_REQUESTED** [review]

**Review by miyurugunarathna — APPROVED** [review]
LGTM

**Review by senuravihanjayadeva — APPROVED** [review]

**Pull request merged**